cmd/update.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 The Okteto Authors
```

Comment:
```suggestion
// Copyright 2021 The Okteto Authors
```

Reply:
Should we change all the copyright from 2020 to 2021?

Reply:
yeah, but it's probably better to do it on a separate PR?

Reply:
I think its going to be better to have a single PR for that specific task